### PR TITLE
Signup signin

### PR DESCRIPTION
**Pull request by Sharmaryan**
**functionality**

- added auth
- signup
- signin

things to resolve:
sidebar is still showing up on signup/signin page

### CODE REVIEW

**Review by agrawalshanu708 — COMMENTED** [review]
code looks clean.
UI suggestion:
give proper spacing in card text
make card height less

**Review by shashikantmaurya0007 — APPROVED** [review]
code looks clean.
UI suggestion:
give proper spacing in card text
make card height less

**Comment by raghav6202** [comment]
Could place the log in component at the center, it might look better. You can give some padding to the Title as it is too much to the left. Take care of placeholder spacing too.

Apart from these things  looks good!

File: src/App.js [hunk 1]
```diff
@@ -3,19 +3,22 @@ import "./App.css";
...
       <Routes>
-        <Route path="/" element={<Homepage />} />
+        <Route path="/" element={<SigninPage />} />
+        <Route path="/signup" element={<SignupPage />} />
+        <Route path="/home" element={<Homepage />} />
         <Route path="/archive" element={<Archive />} />
         <Route path="/trash" element={<Trash />} />
         <Route path="/profile" element={<Profile />} />
         <Route path="/labels" element={<Labels />} />
+        <Route path="/mock" element={<Mockman />} />
```
Comment (agrawalshanu708) [inline src/App.js:21]: Make a folder of seperate route, and import from them

**Review by agrawalshanu708 — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (Sharmaryan) [reply src/App.js:21]: sure,  I will do that

**Review by Himanshu-Dhawale — COMMENTED** [review]
UI looks good.
Code also looks good.
I will suggest to make the changes suggested by Shashi

**Pull request merged**